src/components/SectionDesignTools.js

```diff
@@ -8,19 +8,30 @@ import clock from '../images/clock.svg';
 import icon from '../images/icon.svg';
 import heart from '../images/heart.svg';
 import Button from './Button';
+import Container from './Container';
 
 const Wrapper = styled.div`
   background: url(${background}) no-repeat center bottom;
-  margin-bottom: 110px;
+  margin-bottom: 30px;
+
+  @media screen and (min-width: 450px) {
+    margin-bottom: 60px;
+  }
+
+  @media screen and (min-width: 850px) {
+    margin-bottom: 100px;
+  }
 `;
 
-const Container = styled.div`
+const ContainerStyled = styled(Container)`
   display: flex;
   flex-direction: column;
   align-items: center;
-  max-width: 1130px;
-  margin: 0 auto;
-  padding-bottom: 90px;
+  padding-bottom: 50px;
+
+  @media screen and (min-width: 660px) {
+    padding-bottom: 90px;
+  }
 `;
 
 const NewH2 = styled(H2)`
```

Comment:
The naming of this styled component isn't good.  Use something more meaningful for this specific section e.g. `Heading` or `SectionHeading`

Reply:
Done